src/neuroconv/datainterfaces/ecephys/spikegadgets/spikegadgetsdatainterface.py

```diff
@@ -22,6 +22,7 @@ def get_source_schema(cls) -> dict:
         source_schema["properties"]["file_path"].update(description="Path to SpikeGadgets (.rec) file.")
         return source_schema
 
+    # @validate_call
```

Comment:
Some particular lines have an issue that might be related to the way the current ArrayType is defined; I'm leaving them as commented out and will debug in the next followup